dist/styles/RecipeTile.css

```diff
@@ -24,7 +24,7 @@
 .recipeTileHeader {
   display: flex;
   align-items: center;
-  font-size: 250%;
+  font-size: 180%;
```

Comment:
Why is this in % and not em?

Reply:
I adjusted the font size for the recipeTile because I thought it's too big. 
I guess we can talk about this tomorrow with @leonardodeutsch to change it to em.